internal/authctx/helper.go

```diff
@@ -76,14 +82,45 @@ func ProviderAuthSchema() map[string]*schema.Schema {
 	}
 }
 
+var selfManagedAuthSchema = &schema.Schema{
```

Comment:
Docs regeneration is needed post addition/ update of any schema to be reflected in the generated schema.

Reply:
Yes, documentation would taken in different task